upgrade_test.py

```diff
@@ -828,6 +831,129 @@ def test_rolling_upgrade(self):  # pylint: disable=too-many-locals,too-many-stat
 
         InfoEvent(message='all nodes were upgraded, and last workaround is verified.').publish()
 
+    def test_custom_profile_rolling_upgrade(self):  # pylint: disable=too-many-locals,too-many-statements
+        """
+        Run a load of a custom profile.
+        Upgrade half of nodes in the cluster, and run rollback.
+        Upgrade all nodes to new version in the end.
+        """
+        InfoEvent(message='Running a prepare load for the initial custom data').publish()
+        prepare_cs_user_profiles = self.params.get('prepare_cs_user_profiles')
+        stress_before_upgrade, _ = self.run_cs_user_profiles(cs_profiles=prepare_cs_user_profiles, duration="15m")
+        for queue in stress_before_upgrade:
+            self.verify_stress_thread(cs_thread_pool=queue)
+
+        # write workload during entire test
+        InfoEvent(message='Starting write workload during entire test').publish()
+        cs_user_profiles = self.params.get('cs_user_profiles')
+        entire_write_thread_pool, stress_cmds = self.run_cs_user_profiles(cs_profiles=cs_user_profiles, duration="28m")
+
+        if "scylla-qa-internal" in ",".join(cs_user_profiles):
+            stress_cmds = cs_user_profiles
+        self.log.debug("stress_cmds: %s", stress_cmds)
+
+        # Let to write_stress_during_entire_test complete the schema changes
+        # TODO: adapt to use sdcm.utils.user_profile.get_keyspace_from_user_profile to get keyspace name
+        for user_profile in stress_cmds:
+            self.log.debug("user_profile: %s", user_profile)
+            _, profile = get_profile_content(user_profile)
+            keyspace_name = profile.get('keyspace')
+            if not keyspace_name:
+                continue
+            # keyspace_name = "custom_d1"
+            try:
+                self.metric_has_data(
+                    metric_query='sct_cassandra_stress_write_gauge{type="ops", keyspace="%s"}' % keyspace_name, n=10)
```

Comment:
should use `sct_cassandra_stress_user_gauge`

Reply:
fixed